### PR TITLE
fix: post timeline not update

### PR DESCRIPTION
**Pull request by Jack-Works**
Note: this is a workaround (but maybe it is even better than `MutationObserverWatcher`?)

### CODE REVIEW

File: packages/maskbook/src/social-network-adaptor/twitter.com/collecting/post.ts [hunk 1]
```diff
@@ -94,7 +93,8 @@ function registerPostCollectorInner(
                 ? `${isQuotedTweet ? 'QUOTED' : ''}${postIdParser(tweetNode)}${node.innerText.replace(/\s/gm, '')}`
                 : node.innerText
         })
-    startWatch(watcher, cancel)
+    watcher.startWatch(250)
```
Comment (guanbinrui) [inline packages/maskbook/src/social-network-adaptor/twitter.com/collecting/post.ts:96]: What does this mean?

Comment (Jack-Works) [reply packages/maskbook/src/social-network-adaptor/twitter.com/collecting/post.ts:96]: The most used watcher in our project is MutationObserverWatcher which is based on the MutationObserver. It no longer works on Twitter for unknown reason and we have other watchers, now I switch to IntervalWatcher which is based on setInterval.

250 is the interval of how often this watcher will check the state.

Comment (guanbinrui) [reply packages/maskbook/src/social-network-adaptor/twitter.com/collecting/post.ts:96]: How about other selectors? Do they need to use this watcher too?

Comment (Jack-Works) [reply packages/maskbook/src/social-network-adaptor/twitter.com/collecting/post.ts:96]: No idea. If they works just leave it for now.

**Review by guanbinrui — APPROVED** [review]

**Review by jk234ert — APPROVED** [review]

**Pull request merged**